xmodule/tests/test_item_bank.py

```diff
@@ -0,0 +1,498 @@
+        self.item_bank.runtime.publish = self.publisher
+
+
+@skip_unless_cms
+class TestItemBankForCms(ItemBankTestBase):
+    """
+    Test Studio ItemBank behaviors -- export/import, validation, author-facing views.
+    """
+    def test_xml_export_import_cycle(self):
+        """
+        Test the export-import cycle.
+        """
+        # Export self.item_bank to the virtual filesystem
+        export_fs = MemoryFS()
+        self.item_bank.runtime.export_fs = export_fs  # pylint: disable=protected-access
+        node = etree.Element("unknown_root")
+        self.item_bank.add_xml_to_node(node)
+
+        # Read back the itembank OLX
+        with export_fs.open('{dir}/{file_name}.xml'.format(
+            dir=self.item_bank.scope_ids.usage_id.block_type,
+            file_name=self.item_bank.scope_ids.usage_id.block_id
+        )) as f:
+            actual_olx_export = f.read()
+
+        # And compare.
+        expected_olx_export = (
+            '<itembank display_name="My Item Bank" max_count="1">\n'
+            '  <problem url_name="My_Item_0"/>\n'
+            '  <problem url_name="My_Item_1"/>\n'
+            '  <problem url_name="My_Item_2"/>\n'
+            '  <problem url_name="My_Item_3"/>\n'
+            '</itembank>\n'
+        )
+        self.maxDiff = None
+        assert actual_olx_export == expected_olx_export
+        olx_element = etree.fromstring(actual_olx_export)
+
+        # Re-import the OLX.
+        runtime = TestImportSystem(load_error_blocks=True, course_id=self.item_bank.location.course_key)
+        runtime.resources_fs = export_fs
+        imported_item_bank = ItemBankBlock.parse_xml(olx_element, runtime, None)
+
+        # And make sure the result looks right.
+        self._verify_xblock_properties(imported_item_bank)
+
+    def test_xml_import_with_comments(self):
+        """
+        Test that XML comments within ItemBankBlock are ignored during the import.
+        """
+        # Export self.item_bank to the virtual filesystem
+        export_fs = MemoryFS()
+        self.item_bank.runtime.export_fs = export_fs  # pylint: disable=protected-access
+        node = etree.Element("unknown_root")
+        self.item_bank.add_xml_to_node(node)
+
+        # Now, import the itembank using the same OLX as above, but with some XML comments in there.
+        # (Note: This will not work without exporting first, because it relies on the problem blocks OLX definitions
+        # being available on the filesystem)
+        olx_with_comments = (
+            '<!-- Comment -->\n'
+            '<itembank display_name="My Item Bank" max_count="1">\n'
+            '<!-- Comment -->\n'
+            '  <problem url_name="My_Item_0"/>\n'
+            '  <problem url_name="My_Item_1"/>\n'
+            '  <problem url_name="My_Item_2"/>\n'
+            '  <problem url_name="My_Item_3"/>\n'
+            '</itembank>\n'
+        )
+        olx_element = etree.fromstring(olx_with_comments)
```

Comment:
As far as I know, this function will just skip over comments, so I'm not sure what this test is actually testing that's any different than if the comments didn't exist. We don't need to verify that `etree` can ignore comments when parsing XML.

Reply:
Yeah, true. This test case also exists in test_library_content.py too, all the more reason to omit it here. Removed.